prototypes/Buildings.lua

```diff
@@ -712,13 +712,17 @@ function buildings.enable()
         airFilter.fixed_recipe = "air-filter-pollution-rampant-industry"
         airFilter.crafting_categories = { "air-filter-rampant-industry" }
         airFilter.minable = {mining_time=2,result="air-filter-rampant-industry"}
+        airFilter.icon = "__RampantIndustry__/graphics/icons/air-filter.png"
+        airFilter.icon_size = 64
+        airFilter.icon_mipmaps = 1
         airFilter.next_upgrade = nil
         airFilter.energy_usage = "750KW"
-        airFilter.energy_source.drain = "500KW"
+        airFilter.energy_source.drain = "50KW"
```

Comment:
Why a 10x reduction? Are expecting 10x more of these to be built any balancing on the outputs of the pollution processing recipe?

Reply:
That's only drain, NOT energy consumption in working stage. Quote from mod discussion thread:
"500/1500KW is quite a lot for doing nothing."

If you think that drain must be bigger... it's up to you. But I think it should be in energy_usage. For example increase it from 750KW to 1150KW. Drain occurs when:
a) no filters in ingredients slot - no filters, no work,
b) not enough energy,
c) not enough pollution,
d) what I added in script - protected state for marginal amount of pollution.

Am I wrong?

Reply:
sounds reasonable, but I haven't had time to verify

Reply:
sounds reasonable, but I haven't had time to verify

Reply:
That is why I did spend time to verify it for you! :-)